include/saslplug.h

```diff
@@ -153,7 +153,10 @@ typedef struct sasl_utils {
     void (*prop_erase)(struct propctx *ctx, const char *name);
     int (*auxprop_store)(sasl_conn_t *conn,
 			 struct propctx *ctx, const char *user);
-
+    /* Registry value functions */
+    int (*get_registry_value)(void *context __attribute__((unused)),
```

Comment:
Is this part of the liblsasl ABI? If so, soname (`sasl_version` in `lib/Makefile.am`) would need bumping...

Reply:
True